package.json

```diff
@@ -19,7 +19,9 @@
     "express": "^4.16.2",
     "mongoose": "^4.13.6",
     "react-css-modules": "4.7.1",
+    "react-geolocated": "^2.2.1",
     "react-google-maps": "^9.4.3",
+    "react-places-autocomplete": "^5.4.3",
```

Comment:
We should take out the '^' before each version because that's a signal to automatically update to the latest version, and can sometimes break code